test/functional/test-cid-api.js

```diff
@@ -126,6 +126,28 @@ describes.realWin('test-cid-api', {}, env => {
     });
   });
 
+  it('should try alternative url if API provides', () => {
+    fetchJsonStub.onCall(0).returns(Promise.resolve({
+      json: () => {
+        return {
+          alternateUrl: 'https://ampcid.google.uk/v1/publisher:getClientId',
```

Comment:
nit, i don't think this tld exists. doesn't really matter for your test, but maybe change to .co.uk.

Reply:
done : )